docs/topic-guides/blendable-web-apps/view-composing.md

```diff
@@ -63,6 +63,16 @@ The key for the custom compositions is the main view. If there is more than one
 
 ![](../../.gitbook/assets/subset-match.gif)
 
+### Glossary
+
+This file uses the following definitions:
+
+- *view* - URI of the partial HTML view file in a Starcounter app
+- *main view* - view of the response for the browser or `Self.GET` request
+- *attached view* - views that extend the main view through blending
+- *custom composition* - composition stored in the database tables `Starcounter.HTMLComposition` (legacy, deprecated) or `Starcounter.HtmlViewComposition` + `Starcounter.HtmlViewAttachment` + `Starcounter.HtmlView` (current)
+- *default composition* - composition provided by the app author in `declarative-shadow-dom` of the HTML file
+
```

Comment:
I am not convinced this is absolutely necessary in this form. It would be better to spread the definitions where they are needed.

*View* is defined in https://docs.starcounter.io/v/2.4/topic-guides/blendable-web-apps/html-views, no need to redefine it here.

*Main* and *attached view* should be defined in the "Introduction" section, as a  last paragraph:

> In a composition of multiple views, there is always a single **main view** and one or more **attached views**. The main view comes from the app that responded to a URI request from a web browser or from a `Self.GET`. The attached views are added to the main view based on the [attachment rules](view-model-attaching.md).

*Default composition* should be explained as the first paragraph of "Default compositions of the attached views":

> Default compositions of HTML views are provided by the app authors in the `declarative-shadow-dom` section of the HTML file. The guidelines for authoring compositions are presented in [HTML Views Composition Guidelines](html-views-composition-guidelines.md).

*Custom composition* should be explained as the first paragraph of "Custom compositions of the attached views":

> Custom compositions are created by solution owners and stored as configuration in the database. A solution owner can manage custom compositions using [BlendingEditor](https://github.com/Starcounter/Blending). Custom compositions are stored in the system tables `Starcounter.HTMLComposition` (legacy, deprecated) or `Starcounter.HtmlViewComposition` + `Starcounter.HtmlViewAttachment` + `Starcounter.HtmlView` (current).
-----

I also think that the section "Subset and superset composition matching" should be the last section before "Summary", because that's only interesting for advanced users.